packages/workflow/src/internals.ts

```diff
@@ -1,4 +1,6 @@
 import { PayloadConverter } from '@temporalio/common';
+import type { RawSourceMap } from 'source-map';
+import errorStackParser = require('error-stack-parser');
```

Comment:
You can use import from syntax but I wouldn't bother, I'm going to do some refactoring of this code